service/history/worker_versioning_util.go

```diff
@@ -196,7 +197,7 @@ func initializeWorkflowAssignedBuildId(
 		return err
 	}
 
-	if mutableState.HasCompletedAnyWorkflowTask() {
+	if mutableState.HasCompletedAnyWorkflowTask() || workflowTask.StartedEventID != common.EmptyEventID {
```

Comment:
Is this change supposed to be in this PR?

Reply:
just for now, since I was using this draft-action to test that change. needs to be reverted before merging. I didn't expect people to review this since it's marked as a draft